test/integration/test-cases/include-template-func/input/helmfile.yaml

```
@@ -0,0 +1,12 @@
+releases:
+- name: {{ include "echo" (dict "Echo" "include") }}
+  chart: ../../../charts/raw
+  values:
+  - templates:
+    - |
+      apiVersion: v1
+      kind: ConfigMap
+      metadata:
+        name: {{ include "echo" (dict "Echo" "include") }}
+      data:
+        name: {{ include "echo" (dict "Echo" "include") }} 
```

Maybe you could also cover sub-helmfiles in this test case to confirm https://github.com/helmfile/helmfile/pull/1187/files#r1412908641 !

```suggestion
        name: {{ include "echo" (dict "Echo" "include") }} 
helmfiles:
- nested/helmfile.yaml
```

And have:

- `nested/_helpers.tpl`
  ```
  {{- define "echo" -}}
  nested/{{ .Echo }}
  {{- end }}
  ```
- `nested/helmfile.yaml`
  - ```
    releases:
    - name: {{ include "echo" (dict "Echo" "include") }}
      chart: ../../../charts/raw
      values:
      - templates:
        - |
          apiVersion: v1
          kind: ConfigMap
          metadata:
            name: {{ include "echo" (dict "Echo" "include") }}
          data:
            name: {{ include "echo" (dict "Echo" "include") }} 
     ```

This way, the nested helmfile.yaml would render `nested/include` if it didn't conflate the `_helpers.tpl` files in the different directories?

---

currently. it only loads _helper.tpl in the root dir. @mumoshu